app/scripts/SeriesListMenu.js

```diff
@@ -19,6 +19,20 @@ export default class SeriesListMenu extends ContextMenuContainer {
   getConfigureSeriesMenu(position, bbox, track) {
     const menuItems = {};
 
+    // plugin tracks can offer their own options
+    // if they clash with the default higlass options
+    // they will override them
+    const pluginOptionsInfo =
+      window.higlassTracksByType &&
+      window.higlassTracksByType[track.type] &&
+      window.higlassTracksByType[track.type].config.optionsInfo;
```

Comment:
Is `window.higlassTracksByType[track.type].config` always available? If not this could lead to issues when `window.higlassTracksByType[track.type]` is available but `window.higlassTracksByType[track.type].config` is not. Simple fix:

```
const pluginOptionsInfo =
      window.higlassTracksByType &&
      window.higlassTracksByType[track.type] &&
      window.higlassTracksByType[track.type].config &&
      window.higlassTracksByType[track.type].config.optionsInfo;
```

Reply:
It probably will always be available but it doesn't hurt to check.